pyproject.toml

```diff
@@ -47,7 +47,7 @@ backoff = ">=2.0.0,<3.0.0"
 SPARQLWrapper = { version = "^2.0.0", optional = true }
 pyodbc = { version = "~4.0.32", optional = true }
 modin = { version = "~0.16.0", optional = true }
-ray = { version = "~2.0.0", extras = ["default", "data"], optional = true}
+ray = { version = "^2.2.0", extras = ["default", "data"], optional = true}
```

Comment:
Hmm the only risk here is that the Glue team does not upgrade as well
What is the risk of allowing `>=2.0, <2.3.0`?

Reply:
We want at least `2.2` so that we can get PyArrow 7+. If we don't do that, we'll have a lot of difficulties merging the `deltalake` feature.

In terms of `<2.3.0`, do we have a good reason to block future minor versions of Ray?

Reply:
But I would have thought that even with ray 2.0 to 2.2, if someone tries to install delta lake 6+, the pip resolver would realize that it needs to choose ray 2.2 to enable pyarrow 7 no?

Reply:
I think we import some private ray modules and we are worried that an upgrade would break that if they were to modify them